tasks/filter-team-data/test/test_team_data_filterer.rb

```diff
@@ -13,7 +13,7 @@
       ]
 
       results = TeamDataFilterer.transform_links(links)
-      expect(results).must_equal([
+      results.must_equal([
```

Comment:
I think they've been failing (they're failing in master now) since they were switched from Rspec -> MiniTest(MiniSpec).   The change is to MiniTest format.